source/sass/blocks/form.scss

```
@@ -60,6 +95,16 @@
   background-color: #81b3d2;
 }
 
+.form__button:hover,
+.form__button:focus {
+  background-color: #669ec0;
+}
+
+.form__button:active {
+  background-color: #5496bd;
+  color: rgba(255, 255, 255, 0.3);
+}
```

Для подобных правил удобно использовать вложенность
```suggestion
.form__button {
  &:hover,
  &:focus {
    background-color: #669ec0;
  }
  
  &:active {
    background-color: #5496bd;
    color: rgba(255, 255, 255, 0.3);
  }
}
```